app/regen/cmd/cmd_test.go

```diff
@@ -1,23 +1,25 @@
 package regen_test
 
 import (
-	"fmt"
+	"io/ioutil"
 	"testing"
 
+	svrcmd "github.com/cosmos/cosmos-sdk/server/cmd"
 	"github.com/stretchr/testify/require"
 
-	"github.com/cosmos/cosmos-sdk/x/genutil/client/cli"
 	cmd "github.com/regen-network/regen-ledger/app/regen/cmd"
 )
 
 func TestInitCmd(t *testing.T) {
+	nodeHome, err := ioutil.TempDir(t.TempDir(), ".regen")
+	require.NoError(t, err)
+
 	rootCmd, _ := cmd.NewRootCmd()
 	rootCmd.SetArgs([]string{
 		"init",          // Test the init cmd
 		"regenapp-test", // Moniker
-		fmt.Sprintf("--%s=%s", cli.FlagOverwrite, "true"), // Overwrite genesis.json, in case it already exists
```

Comment:
Hmm I don't think it hurts. There could be edge cases where is needed.

Reply:
What edge cases are you thinking of? Ideally this should be run as a self contained test with full teardown of any artifacts right? The home directory is created from scratch on each test run, so I don't see how we could expect there to already be a `genesis.json` here.

Previously this line was there because it always looked at your default home directory for running (or created one if it didnt exist). This PR changes that so it creates a temp home dir, which should render the `--overwrite=true` flag unnecessary. Lmk if i'm missing something